python/ray/autoscaler/_private/fake_multi_node/test_utils.py

```diff
@@ -240,3 +245,66 @@ def stop(self):
             f"RAY_FAKE_CLUSTER=1 ray down -y {self.config_file}", shell=True)
 
         self._stop_monitor()
+
+    def _update_nodes(self):
+        with open(self._nodes_file, "rt") as f:
+            self._nodes = json.load(f)
+
+    def _update_status(self):
+        with open(self._status_file, "rt") as f:
+            self._status = json.load(f)
+
+    def _get_node(self,
+                  node_id: Optional[str] = None,
```

Comment:
Nit: For me `num` and `rand` would be necessary. Not sure if I would use `node_id`.

Reply:
`num` is just a wrapper, but `node_id` can be useful if people fetch node information (e.g. object locations are given with node ids)